src/app/dashboard/components/dashboard-visits/dashboard-visits.component.html

```diff
@@ -1 +1,75 @@
-<p>dashboard-visits works!</p>
+<h2>
+  <strong>{{ total }}</strong>
+  {{ 'strings.headingVisits' | translate }}
+</h2>
+
+<ul *ngIf="data" class="visits">
+  <li class="visits-unavailable">
+    <span class="count">{{ data.visits.UNAVAILABLE ?? 0 }}</span>
+    <span class="statistic down" *ngIf="unavailablePercent < 0"
+      ><mat-icon class="material-icons-round icon">keyboard_arrow_down</mat-icon></span
+    >
+    <span class="statistic up" *ngIf="unavailablePercent > 0"
+      ><mat-icon class="material-icons-round icon">keyboard_arrow_up</mat-icon></span
+    >
+    <span class="statistic equal" *ngIf="unavailablePercent === 0"
+      ><mat-icon class="material-icons-round icon">remove</mat-icon></span
+    >
+    <span class="percentage">{{ unavailablePercent | percent }}</span>
+    <mat-icon class="material-icons-round icon">block</mat-icon>
+    <span class="info">{{ 'captions.dashboardUnavailable' | translate }}</span>
+    <mat-progress-bar [value]="(data.visits.UNAVAILABLE / total) * 100"></mat-progress-bar>
```

Comment:
you could use 'unavailablePercent * 100' since it's already computed and save additional operations
same for the rest

Reply:
done.